packages/mdc-dialog/README.md

```diff
@@ -89,8 +89,9 @@ const dialog = new MDCDialog(document.querySelector('.mdc-dialog'));
 > See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
 
 MDC Dialog makes no assumptions about what will be added to the `mdc-dialog__content` element. Any List, Checkboxes,
-etc. must also be instantiated. Additionally, call `layout` on any applicable components within the content when
-`MDCDialog:opened` is emitted.
+etc. must also be instantiated. If your dialog contains any layout-sensitive components, you should wait until
```

Comment:
It might be clearer if we include an example of what we mean by "layout-sensitive". Something like:

```markdown
If your dialog contains any layout-sensitive components
(e.g., some elements appear mispositioned or incorrectly sized),
you should wait until the `MDCDialog:opened` event is emitted to instantiate them
(or call `layout()` on them) so that the dialog's transition finishes first.
```

or:

```markdown
If your dialog contains any layout-sensitive components,
you should wait until the `MDCDialog:opened` event is emitted to instantiate them
(or call `layout()` on them) so that the dialog's transition finishes first.
Mispositioned or incorrectly-sized elements (e.g., ripples or floating labels)
are a strong indication that child components are being instantiated
before the dialog has finished opening.
```

Reply:
Good idea. I added it as a NOTE after the code snippet instead to avoid elongating the prose here.